python/cuml/test/test_fil.py

```diff
@@ -491,8 +491,9 @@ def test_output_args(small_classifier_and_preds):
     assert array_equal(fil_preds, xgb_preds, 1e-3)
 
 
-def to_categorical(features, n_categorical):
-    # returns data in two formats: pandas (for LightGBM) and numpy (for FIL)
+def to_categorical(features, n_categorical, invalid_pct, rng):
```

Comment:
It's much better to specify the fraction of invalid categories directly as a floating-point number rather than percentage. E.g. 0.05 instead of 5 (percent).